winfetch.ps1

```diff
@@ -872,7 +872,7 @@ function info_pkgs {
     }
 
     if ("choco" -in $ShowPkgs -and (Get-Command -Name choco -ErrorAction Ignore)) {
-        $chocopkg = (& choco list -l)[-1].Split(' ')[0] - 1
+        $chocopkg = (& choco list)[-1].Split(' ')[0] - 1
```

Comment:
```suggestion
        $chocopkg = if (([version] (& choco --version).Split('-')[0]) -ge [version] '2.0.0') {
            (& choco list)[-1].Split(' ')[0] - 1
        } else {
            (& choco list --local-only)[-1].Split(' ')[0] - 1
        }
```

The fix for supporting Chocolatey CLI v2.x is a little more involved than just dropping the `-l` option, as this code will still need to support v1.x. In the older versions, `choco list` without the `-l` searches remote sources, and so it is still needed when you're running the older choco version.

My suggested edit here checks the version of Chocolatey CLI and adjusts the command as required. In the hopes of increasing readibilty and maintainability, I also expanded out the option to the full `--local-only`.